osmnx/save_load.py

```diff
@@ -373,15 +373,16 @@ def get_undirected(G):
         # if we haven't already flagged this edge as a duplicate
         if not (u, v, key) in duplicate_edges:
 
-            # look at this first edge's parallel edges one at a time
-            for key_other in H.edge[u][v]:
+            # look at every other edge between u and v, one at a time
+            for key_other in dict(H[u][v]).keys():
```

Comment:
The code ```H[u][v]``` creates a dict.  So you don't need the dict/keys parts.  Replace ```G.edge[u][v]``` with either ```G[u][v]``` or ```G.adj[u][v]```   (they are equivalent)